styles/index.css

```
@@ -1,19 +1,35 @@
 /*external sources*/
 @import url("https://fonts.googleapis.com/css2?family=Lalezar&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Noto+Sans+KR:wght@100..900&display=swap");
 @import "animate.css/animate.css";
 ._font-roboto {
   font-family: Roboto sans-serif;
 }
 
+._font-notosans {
+  font-family: Noto Sans KR;
```

Stylelint (17.7.0)

[error] Expected quotes around "Noto Sans KR" (font-family-name-quotes)

(font-family-name-quotes)